user_docs/en/userGuide.t2t

```diff
@@ -2785,10 +2785,19 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 
 ++ Log Viewer ++[LogViewer]
 The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you last started NVDA.

```

Comment:
To avoid any ambiguity with the word "last":

```suggestion
The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred during the current run of NVDA.

```

Reply:
Could you explain why "last" is ambiguous? On the contrary, I feel that the concept of "NVDA start" is clearer than the concept of "NVDA run".

Reply:
My thought was only that "last started" might imply, to some readers, the concept of "previously started".

Perhaps another rephrasing might be:

```suggestion
The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you most recently started NVDA.

```

Reply:
I tried to change this, hope this satisfies?